tests/integration/200.command.dev.test.cjs

```diff
@@ -226,15 +230,21 @@ export const handler = async function () {
         })
         .withEdgeFunction({
           handler: async (req, { next }) => {
-            if (!req.url.includes('?ef=true')) {
-              return
+            if (req.url.includes('?ef=true')) {
```

Comment:
What if the `ef` query param is not the first one? is this possible?
I would go with:

```typescript
switch(new URL(req.url).searchParams.get('ef')) {
  case 'true':
    const res = await next()
    const text = await res.text()
    return new Response(text.toUpperCase(), res)
  case 'fetch':
    const url = new URL('/origin', req.url)
    const res = await fetch(url)
    const text = await res.text()
    return new Response(text.toUpperCase(), res)
}
```